homeassistant/components/transmission/switch.py

```diff
@@ -45,6 +47,12 @@ def __init__(self, switch_type, switch_name, tm_client, name):
         self._state = STATE_OFF
         self._data = None
         self.unsub_update = None
+        self._attr_device_info = DeviceInfo(
+            entry_type=DeviceEntryType.SERVICE,
+            identifiers={(DOMAIN, tm_client.config_entry.entry_id)},
+            manufacturer="Transmission",
+            name=name,
```

Comment:
The device here is named by its switch name, while in the previous instance, the same device (with the same identifier) is named by its configuration name?

Which one is true? A single device can't have different names :)

Reply:
Both represent the same thing. For some reason `TransmissionSwitch` constructor uses `name` variable instead of `client_name`